examples/vep_data/tools_and_consequence.jsonc

```diff
@@ -0,0 +1,541 @@
+                    "license": {
+                        "type": "In Silico Tool License",
+                        "primaryCoding": {
+                            "system": "https://s3.amazonaws.com/sift-public/nsSNV/sift6.2.1.tar.gz",
+                            "code": "FHCRC Noncommercial License" 
+                        }
+                    },
+                   "reportedIn": {
+                        "type": "Document",
+                        "title": "Predicting deleterious amino acid substitutions",
+                        "urls": [
+                            "https://pubmed.ncbi.nlm.nih.gov/11337480/",
+                            "https://s3.amazonaws.com/sift-public/nsSNV/sift6.2.1.tar.gz"
+                        ],
+                        "doi": "10.1101/gr.176601",
+                        "pmid": 11337480
+                    }
+                }
+            }
+        ],
+        "directionOfEvidenceProvided": "neutral",
+        "strengthOfEvidenceProvided": {
+            // Leaving this blank because 0.06 does not fall into any of the buckets in the ACMG paper. Thus direction is set to neutral...
+            // You'll notice this DOESN'T correspond to the categorical label SIFT provides. 
+        },
+        "evidenceOutcome": {
+            "conceptType": "SIFT Outcome",
+            "primaryCode": "tolerated"
+        },
+        "focusVariant": "#/input_variant"
+    },
+    "spliceAI": {
+        "type": "InSilicoVariantEffectPredictionEvidenceLine",
+        "targetProposition": "#/insilico_tool_splice_proposition",
+        "hasEvidenceItems": [
+            {
+                "type": "InSilicoVariantEffectScoreStudyResult",
+                "label": "SpliceAI Donor Gain Score",
+                "focusVariant": "#/input_variant",
+                "transcripts": [
+                    // IRI references to the transcript molecules from the molecular consequence study result
+                    "#/missense_mol_con_1/transcriptMolecule",
+                    "#/missense_mol_con_2/transcriptMolecule"
+                ],
+                "regulatoryFeatures": [], // Tool scores can also be w.r.t. regulatory features
+                "score": 0,
+                "ancillaryResults": {
+                    "deltaPosition": -29
+                },
+                "specifiedBy": {
+                    "type": "Method",
+                    "methodType": "Algorithmic Output",
+                    "reportedIn": {
+                        "type": "Document",
+                        "title": "Predicting Splicing from Primary Sequence with Deep Learning",
+                        "urls": ["https://pubmed.ncbi.nlm.nih.gov/30661751/"],
+                        "doi": "10.1016/j.cell.2018.12.015",
+                        "pmid": 30661751
+                    }
+                },
+                "sourceDataSet": {
+                    "type": "DataSet",
+                    "datasetType": "SpliceAI",
```

Comment:
It would be useful to hold additional attribs eg the transcript set used for predictions and whether the scores are masked or raw